main/zoe/api/Zoe-Intro.md

```diff
@@ -19,32 +19,41 @@ This is an introduction to Zoe, Agoric's smart contract framework. The Zoe Guide
 To start using a contract, you need to *install* it on Zoe. This is putting the contract’s code into Zoe. Multiple contracts can be installed on Zoe, and each contract can have multiple simultaneous active offers, each of which is handled by a separate contract instance. For clarity, in this document a *contract* is the installed code that defines how the contract works. A *contract instance* is an active execution of the contract code.
 
 
-Once installed on Zoe, a contract waits for an *offer* to be made. That starts a lifecycle for the offer and new contract instance of:
-1. Sending out *invites* to request counter-offers.
-2. Partie *redeeming* those invites and making a counter-offer.
-3. Determining when the offer *matches* a counter-offer.
-4. Either succeeding or failing to make the transaction. 
+Once installed on Zoe, a contract is instantiated, staring a lifecycle new contract instance of:
+1. Instantiating the contract with
+   ```js
+   const initialInviteP = zoe~.makeInstance(
+     installationHandle,
+     issuerKeywordRecord,
+     terms);
```

Comment:
Our eslint settings would insist on a trailing comma and newline before the close paren. Do you want it to be relatively pretty, or to reflect our style?

Reply:
We shouldn't use ~ now in our docs, since we're not yet suggesting it to users, because we don't yet have linting set up correctly in at least VSCode